Common/UnitDefinitions/Length.json

```diff
@@ -398,6 +398,20 @@
           "Abbreviations": [ "ch" ]
         }
       ]
+    },
+    {
+      "SingularName": "Angstrom",
+      "PluralName": "Angstroms",
+      "FromUnitToBaseFunc": "{x}/1e-10",
```

Comment:
@angularsen I have been using spaces in the functions to match the styling of the rest of the code. Would you agree that the following is more consistent? 
```json
"FromUnitToBaseFunc": "{x} / 1e-10",
"FromBaseToUnitFunc": "{x} * 1e-10",
```

Reply:
I'm agree with you, your example is more consistent. 
Also I didn't the follow the style guideline, other units are spaced like your example in the rest of the file.

Reply:
Yeah I would have to do a find/replace on all the json files to make it consistent with the rest of the code and generated code. 

For example the generated prefixes have a lot of inconsistency:
```csharp
unitConverter.SetConversionFunction<Length>(LengthUnit.Meter, LengthUnit.Nanometer, quantity => new Length((quantity.Value) / 1e-9d, LengthUnit.Nanometer));
unitConverter.SetConversionFunction<Length>(LengthUnit.Meter, LengthUnit.NauticalMile, quantity => new Length(quantity.Value/1852, LengthUnit.NauticalMile));
```

Consistent would look like:
```csharp
unitConverter.SetConversionFunction<Length>(LengthUnit.Meter, LengthUnit.Nanometer, quantity => new Length((quantity.Value) / 1e-9d, LengthUnit.Nanometer));
unitConverter.SetConversionFunction<Length>(LengthUnit.Meter, LengthUnit.NauticalMile, quantity => new Length(quantity.Value / 1852, LengthUnit.NauticalMile));
```

Reply:
And I'm confused about backwards : I followed [this guide](https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit) and I saw this example : 

> Multiply for FromBaseToUnit and divide for FromUnitToBase, so that Length.Centimeter is defined as "FromBaseToUnit": "x*100" and "FromUnitToBase": "x/100" where base unit is Meter

It is an error in the guide or I was on the right way ?

Reply:
> @angularsen I have been using spaces in the functions to match the styling of the rest of the code. Would you agree that the following is more consistent?
> 
> ```json
> "FromUnitToBaseFunc": "{x} / 1e-10",
> "FromBaseToUnitFunc": "{x} * 1e-10",
> ```

Sure, we can do that. We should update existing files to make it consistent and update the wiki guide too then.

Reply:
> > @angularsen I have been using spaces in the functions to match the styling of the rest of the code. Would you agree that the following is more consistent?
> > ```json
> > "FromUnitToBaseFunc": "{x} / 1e-10",
> > "FromBaseToUnitFunc": "{x} * 1e-10",
> > ```
> 
> Sure, we can do that. We should update existing files to make it consistent and update the wiki guide too then.

Will do

Reply:
@JasLieb I updated the wiki to maybe help clarify.
![image](https://user-images.githubusercontent.com/787816/153615142-0a9a90a8-f0fd-49b7-bf25-82bd54e7daa8.png)

@tmilnthorp's point was that the functions are reversed. You would typically see this in the test 

```cs
 protected override double AngstromsInOneMeter => 1e-10;
```

Does one meter equal 1e-10 angstroms?

Reply:
> @JasLieb I updated the wiki to maybe help clarify. ![image](https://user-images.githubusercontent.com/787816/153615142-0a9a90a8-f0fd-49b7-bf25-82bd54e7daa8.png)
> 
> @tmilnthorp's point was that the functions are reversed. You would typically see this in the test
> 
> ```cs
>  protected override double AngstromsInOneMeter => 1e-10;
> ```
> 
> Does one meter equal 1e-10 angstroms?

1 meters = 1e+10 Angstrom, So the overriden field should be :
`protected override double AngstromsInOneMeter  => 1e10;`

If I change and if I apply @tmilnthorp review, I'm back on my feet, sound good to me in this way.